src/main.rs

```diff
@@ -1,8 +1,8 @@
 use getopts::Options;
-use std::{collections::HashMap, error::Error, io::Write, num::ParseIntError, ops::AddAssign};
+use std::{collections::HashMap, error::Error, fs, io::Write, num::ParseIntError, ops::AddAssign};
 
 fn main() -> Result<(), Box<dyn Error>> {
-    let mut opts = getopts::Options::new();
+    let mut opts = Options::new();
```

Comment:
I already wrote a comment about it, but maybe it got lost. I think `Options` is a too generic name for the struct. So I prefer `getopts::Options`. Otherwise it might be less clear, what is happening here.

Reply:
:wrench: :heavy_check_mark: